src/main/scala/nl.knaw.dans.easy.dd2d/DansDeposit2ToDataverseApp.scala

```diff
@@ -77,4 +87,17 @@ class DansDeposit2ToDataverseApp(configuration: Configuration) extends DebugEnha
   def stop(): Try[Unit] = {
     inboxWatcher.stop()
   }
+
+  private def initOutboxDirs(outboxDir: Path): Try[Unit] = Try {
+    if (!File(outboxDir).isEmpty)
+      throw NonEmptyOutboxDirException(outboxDir.toString)
+
+    File(outboxDir.toString.concat(PROCESSED.toString)).createDirectoryIfNotExists(true)
+    File(outboxDir.toString.concat(REJECTED.toString)).createDirectoryIfNotExists(true)
+    File(outboxDir.toString.concat(FAILED.toString)).createDirectoryIfNotExists(true)
+  }
+
+  private def deleteEmptyDepositsDir(inbox: File): Try[Unit] = Try {
```

Comment:
@ChrisMuller80 : Not part of the specs. Same comment as before. If you have ideas for improvement of the specs, that's fine, but don't just build it without any comment. 

In this case I disagree with the extra feature. I think it is confusing if the inbox disappears after a successful run.